lib/internal/bootstrap/loaders.js

```diff
@@ -270,29 +259,29 @@
       const cache = codeCacheHash[id] &&
         (codeCacheHash[id] === sourceHash[id]) ? codeCache[id] : undefined;
 
-      // (code, filename, lineOffset, columnOffset
-      // cachedData, produceCachedData, parsingContext)
-      const script = new ContextifyScript(
-        source, this.filename, 0, 0,
-        cache, false, undefined
+      const fn = compileFunction(
+        source,         // source code
+        this.filename,  // filename
+        0,              // line offset
+        0,              // column offset
+        cache,          // cached data
+        false,          // produce cached data
+        undefined,      // parsing context
+        undefined,      // context extensions
```

Comment:
Missing a comment about the last argument here?

Reply:
I thought it was pretty self-explanatory plus was long, will add one just to be sure.